NEWS

```diff
@@ -112,6 +114,14 @@ Bug fixes
   CIGAR string as an error.  Thanks to Chang Y for reporting the issue.
   (#1298, fixes samtools/samtools#1445)
 
+* Bug fixes in the bundled copy of htscodecs:
+
+  - Fixed an uninitialized access in the name tokenizer decoder.
+    (samtools/htscodecs#23)
+
+  - Fixed a bug with name tokeniser and variable number of names per slice,
+    causing it to incorrectly report an error on certain valid inputs.
+    (samtools/htscodecs#24)
 
```

Comment:
Usually two blank lines before the previous release.

Reply:
Adjusted...

Reply:
Sorry for the picky comments :)  As a penance I have checked out the PR to run a spelling checker on it, and noticed it's got `tokeniser` spelt once each way (s vs z)…

Reply:
All part of our drive for greater inclusivity ;-)

Reply:
Oops, yes.  It should use `tokeniser` I guess, as that's the spelling in the library.